assets/alertmanager/alertmanager.yaml

```diff
@@ -16,11 +16,11 @@ spec:
       preferredDuringSchedulingIgnoredDuringExecution:
       - podAffinityTerm:
           labelSelector:
-            matchExpressions:
-            - key: alertmanager
-              operator: In
-              values:
-              - main
+            matchLabels:
```

Comment:
What happened here? I wonder why it is moving to `matchLabels` instead of `matchExpression`?

Reply:
I updated it in the upstream anti-affinity addon. It now reuses the labelSelector of alertmanager: https://github.com/prometheus-operator/kube-prometheus/commit/4c6a06cf7e496b3f9f752efec7ad214c0b9af169